packages/uma-auth-client/src/steps/ConnectedWallet.tsx

```diff
@@ -1,3 +1,217 @@
+      },
+      {
+        type: PermissionType.READ_TRANSACTIONS,
+        description: "Read transactions",
+      },
+    ],
+    amountInLowestDenom: 50000,
+    amountInLowestDenomUsed: 12345,
+    limitEnabled: true,
+    currency: {
+      code: "USD",
+      name: "US Dollar",
+      symbol: "$",
+      decimals: 2,
+      type: "fiat",
+    },
+    status: ConnectionStatus.ACTIVE,
+    limitFrequency: LimitFrequency.MONTHLY,
+    expiration: "2024-09-30",
+    lastUsed: "2024-09-01",
+  };
+
+  // TODO: Get balance details with NWC connection
+  const balance = {
+    amountInLowestDenom: 50000,
+    currency: {
+      code: "USD",
+      name: "US Dollar",
+      symbol: "$",
+      decimals: 2,
+      type: "fiat",
+    },
+  };
+
+  let limitRenewalString = "";
+  if (connection.limitEnabled) {
+    limitRenewalString = getRenewalString(connection) || "";
+  }
+
+  const limitFrequencyString =
+    connection.limitFrequency !== LimitFrequency.NONE
+      ? `${connection.limitFrequency} spending limit remaining`
+      : "";
+
+  return (
+    <Container>
+      <ConnectionSection>
+        <ConnectionCard uma={uma} connection={connection} balance={balance} />
+        <TextContainer>
+          {balance ? (
+            <Row>
+              <Icon name="Gear" width={16} />
+              <Description>
+                {connection.limitEnabled ? (
+                  <>
+                    <LabelModerate
+                      content={formatAmountString({
+                        amountInLowestDenom: connection.amountInLowestDenomUsed,
+                        currency: connection.currency,
+                      })}
+                    />{" "}
+                    <Label
+                      content={`${limitFrequencyString}${limitFrequencyString && limitRenewalString ? " · " : ""}${limitRenewalString}`}
+                    />
+                  </>
+                ) : (
+                  <LabelModerate content="No spending limit set" />
+                )}
+              </Description>
+            </Row>
+          ) : null}
+          <Row>
+            <Icon name="Clock" width={16} />
+            <Description>
+              {connection.expiration ? (
+                <>
+                  <Label content="Connection expires" />{" "}
+                  <LabelModerate
+                    content={dayjs(connection.expiration).format(
+                      "MMM DD, YYYY",
+                    )}
+                  />
+                </>
+              ) : (
+                <LabelModerate content="No expiration" />
+              )}
+            </Description>
+          </Row>
+        </TextContainer>
+      </ConnectionSection>
+      <ButtonContainer>
+        <Button
+          icon="LinkIcon"
+          text="Manage connection"
+          kind="secondary"
+          externalLink={
+            uma
+              ? `https://nwc.${getUmaDomain(uma)}/connection/${connection.connectionId}`
```

Comment:
I think instead of using nwc.umadomain here, we might need to load this from the uma-configuration too 🤔. Fine for now, but we'll likely need to update that for flexibility.